### ISSUE TITLE
sct_register_multimodal should manage files with the same name but in different folder

### ISSUE THREAD

**Issue by benjamindeleener** [neuropoly/spinalcordtoolbox#373]
When we compute a transformation between two images that have the same name but are located in different folders, sct_register_multimodal correctly register the images but the results are good only in one direction (dest2src).

For example:
image 1: t1.nii.gz
image 2: rescan/t1.nii.gz

The issue is reproducible in sct_issue folder using the following command in sct_issues/20150520_bdeleener/data/:

```
sct_register_multimodal -i seg.nii.gz -d  rescan/seg.nii.gz -p step=1,type=im,algo=affine,metric=MeanSquares,gradStep=0.5,iter=15,smooth=0.5 -x nn
```

**Comment by jcohenadad** [comment]
wow! nice bug :-)

**Comment by benjamindeleener** [comment]
Data are missing, so I tried on `sct_example_data/t2/`. The following commands can be applied to reproduce the error:

```
# preparing data, we only need images.
sct_download_data -d sct_example_data
cd sct_example_data/t2/
mkdir rescan
cp t2.nii.gz rescan/

# here is the real test
sct_register_multimodal -i t2.nii.gz -d rescan/t2.nii.gz -param step=1,type=im,algo=affine,metric=MeanSquares,gradStep=0.5,iter=15,smooth=0.5
```

The problem is that the names of both images are the identical and that the results are produced in the current folder. Therefore, the output in both directions have the same name (in this case: `t2_reg.nii.gz`) and only one direction is kept (dest2src).

How can we solve this issue:
- change the default names of output files (or only one file), for example `t2_destreg.nii.gz`and `t2_srcreg.nii.gz`
- change the output folder? not a good idea...

**Comment by jcohenadad** [comment]
Since this issue happens rarely, I would suggest to check if -i and -d have same name, and if so, to do as Benjamin suggests (t2_dest_reg & t2_src_reg)

**Comment by benjamindeleener** [comment]
Perfect, I implemented this in commit 0fa5fb7e33a44a117cb7b6b7b97c888ac9082adf and integrated in #1072.

**Comment by benjamindeleener** [comment]
Actually caused an error for testing `sct_process_segmentation`: https://travis-ci.org/neuropoly/spinalcordtoolbox/jobs/185224095
